modules/doc/content/getting_started/examples_and_tutorials/tutorial03_verification/step03_analytical_solution.md

```diff
@@ -0,0 +1,247 @@
+
+1. Define a problem, with a known exact solution.
+2. Simulate the problem.
+3. Compute the error between the known solution and simulation.
+4. Perform convergence study.
+
+## Problem Statement
+
+[!cite](incropera1996fundamentals) presents a one-dimensional solution for the heat equation as:
+
+!equation id=tutorial03-analytical-solution
+T(t,x) = T_0 + \frac{\dot{q}}{k} \Bigg\lbrack 2 \sqrt{\frac{\alpha t}{\pi}}
+         \exp\bigg\lparen{\frac{-x^2}{4 \alpha t}}\bigg\rparen -
+         x \bigg\lparen 1-\textrm{erf}\bigg\lparen\frac{x}{2\sqrt{\alpha t}} \bigg\rparen \bigg\rparen \Bigg\rbrack,
+
+where $T$ is the temperature, $T_0$ is the initial temperature, $t$ is time, $x$ is position,
+$\dot{q}$ is the heat source, $k$ is thermal conductivity, and $\alpha$ is the thermal diffusivity.
+
+Thermal diffusivity is defined as the ratio of thermal conductivity ($k$) to the product of density ($\rho$)
+and specific heat capacity ($c$). The system is subjected to Neumann boundary conditions at the
+left boundary ($x=0$) and the natural boundary condition on
+the opposite boundary.
+
+## Simulation
+
+The heat conduction module of [!ac](MOOSE) is capable of performing a simulation of the above problem
+without modification. The values presented in [tutorial03-analytical-values] were selected for the
+simulation and a duration of 1 second shall be utilized.
+
+!table id=tutorial03-analytical-values caption=Numerical values used to the solution values of [tutorial03-analytical-solution].
+| Variable     | Value | Units |
+| :-           | :-    | :-    |
+| $T_0$        | 300   | $K$     |
+| $\dot{q}$    | 7E5   | $W\cdot m^{-3}$ |
+| $k$          | 80.2  | $W\cdot m^{-1}\cdot K^{-1}$ |
+| $\rho$       | 7800  | $kg\cdot m^{-3}$ |
+| $c$          | 450   | $J\cdot kg^{-1}\cdot K^{-1}$ |
+| $q_k(0,t)$   | 7E5   | $W\cdot m^{-2}$ |
+| $q_k(0.03,t)$ | 0    | $W\cdot m^{-2}$ |
+
+### Simulation Setup
+
+The following instructions assumes that there is a working version of [!ac](MOOSE) installed
+on the system, for information on getting it setup please refer to the
+[install instructions](getting_started/installation/index.md).
+
+The simulation to be performed relies on the heat conduction module, thus it is necessary to compile
+that specific module. It is assumed that [!ac](MOOSE) was cloned into a directory named "projects"
+within the home directory. With this assumption, the following commands will compile and test
+the heat conduction module executable.
+
+```bash
+cd ~/projects/moose/modules/heat_conduction
+make -j8
+./run_tests -j8
+```
+
+The number used after the "-j" should be the number of processes available on your system. If tests
+execute without failure, then the following sections should be followed to setup the simulation.
+
+### Simulation Input
+
+[!ac](MOOSE) operates using input files, as such, an input file shall be detailed that is capable of
```

Comment:
this is a lot of shall
it's fine for test specifications, but passive form shalls won't help the non-native readers